py/lexer.c

```diff
@@ -119,6 +125,37 @@ STATIC bool is_tail_of_identifier(mp_lexer_t *lex) {
     return is_head_of_identifier(lex) || is_digit(lex);
 }
 
+STATIC void swap_char_banks(mp_lexer_t *lex) {
+    if (lex->vstr_postfix_processing) {
+        unichar h0, h1, h2;
+
+        h0 = lex->chr0;
+        h1 = lex->chr1;
+        h2 = lex->chr2;
```

Comment:
No need for temporary h variables, can just write directly to chr3-5.

Reply:
Yep - that's now true. A former implementation here used those holding vars (for what, I'm not sure at this point). Good catch.